desktop/src/main/java/bisq/desktop/util/CurrencyList.java

```diff
@@ -36,25 +37,30 @@
 
 import javax.annotation.Nullable;
 
-public class CurrencyList extends ObservableListWrapper<CurrencyListItem> {
+public class CurrencyList {
     private final CurrencyPredicates predicates;
     private final Preferences preferences;
+    private final List<CurrencyListItem> delegate;
 
     public CurrencyList(Preferences preferences) {
         this(new ArrayList<>(), preferences, new CurrencyPredicates());
     }
 
-    CurrencyList(List<CurrencyListItem> delegate, Preferences preferences, CurrencyPredicates predicates) {
-        super(delegate);
+    public CurrencyList(List<CurrencyListItem> delegate, Preferences preferences, CurrencyPredicates predicates) {
+        this.delegate = delegate;
         this.predicates = predicates;
         this.preferences = preferences;
     }
 
+    public ObservableList<CurrencyListItem> getObservableList() {
+        return FXCollections.observableList(delegate);
+    }
+
     public void updateWithCurrencies(List<TradeCurrency> currencies, @Nullable CurrencyListItem first) {
         List<CurrencyListItem> result = Lists.newLinkedList();
         Optional.ofNullable(first).ifPresent(result::add);
         result.addAll(getPartitionedSortedItems(currencies));
-        setAll(result);
+        delegate.addAll(result);
```

Comment:
To have the same behaviour as before (setAll) we should add a clear() before the addAll.